### ISSUE TITLE
Coreference code for RoBERTa

### ISSUE THREAD

**Issue by YeDeming** [allenai/longformer#65]
Thank you for releasing the great Longformer repository! @matt-peters

Could you share the "superhack" code for coreference resolution with mixed PyTorch and Tensorflow implement? Just the raw code without organization is very helpful.

Here is my mailbox, ydm18@mails.tsinghua.edu.cn

Thanks!

Best,
Deming Ye

**Comment by matt-peters** [comment]
The coreference code is running with an older version of a private repo so will take some time to port over. I'll try to release it in the next 1-2 weeks.  I posted a minimal example of passing activations and gradients between pytorch and tensorflow that gives the general idea of how it works in this gist: https://gist.github.com/matt-peters/2c1c4e7eb48eec2029b013935f3c4646

**Comment by YeDeming** [comment]
Thanks for your advice! I will have a try to implement it.
